.env.tpl

```diff
@@ -30,14 +30,12 @@ FEATURE_FLAG_BPMN_SUPPORT=false
 GEMEENTE_CODE=password
 GEMEENTE_MAIL=op://Dimpact/ZAC-.env-$APP_ENV/GEMEENTE/MAIL
 GEMEENTE_NAAM=DummyGemeente
+KEYCLOAK_ADMIN_CLIENT_ID=zaakafhandelcomponent-admin-client
+KEYCLOAK_ADMIN_CLIENT_SECRET=zaakafhandelcomponentAdminClientSecret
```

Comment:
should this also go into the `.env.example`?

Reply:
Sure. This is just a spike so I did not finish the whole thing. :-)